server/svix-server/src/db/mod.rs

```diff
@@ -8,19 +8,24 @@ use crate::cfg::Configuration;
 
 pub mod models;
 
-#[cfg(not(debug_assertions))]
 static MIGRATIONS: sqlx::migrate::Migrator = sqlx::migrate!();
 
-pub async fn init_db(cfg: &Configuration) -> DatabaseConnection {
+async fn connect(cfg: &Configuration) -> sqlx::Pool<sqlx::Postgres> {
     tracing::debug!("DB: Initializing pool");
     if DbBackend::Postgres.is_prefix_of(&cfg.db_dsn) {
-        let sqlx_pool = PgPoolOptions::new().connect(&cfg.db_dsn).await.unwrap();
-
-        #[cfg(not(debug_assertions))]
-        MIGRATIONS.run(&sqlx_pool).await.unwrap();
-
-        SqlxPostgresConnector::from_sqlx_postgres_pool(sqlx_pool)
+        PgPoolOptions::new().connect(&cfg.db_dsn).await.unwrap()
     } else {
         panic!("db_dsn format not recognized. {}", &cfg.db_dsn)
     }
 }
+
+pub async fn init_db(cfg: &Configuration) -> DatabaseConnection {
+    SqlxPostgresConnector::from_sqlx_postgres_pool(connect(cfg).await)
+}
+
+pub async fn run_migrations(cfg: &Configuration) {
```

Comment:
We still want to automatically run it in release mode (for now), so please call it explicitly from `main()`.

The way it is now, it changes behavior.